batchHeader.go

```diff
@@ -179,8 +179,9 @@ func (bh *BatchHeader) Parse(record string) {
 	// 70-75 Date transactions are to be posted to the receivers' account.
 	// You almost always want the transaction to post as soon as possible, so put tomorrow's date in YYMMDD format
 	bh.EffectiveEntryDate = bh.validateSimpleDate(record[69:75])
-	// 76-79 Always blank (just fill with spaces)
-	bh.settlementDate = "   "
+	// 76-78 Always blank if creating batches (just fill with spaces).
+	// Set to file value when parsing. Julian day format.
+	bh.settlementDate = bh.validateSettlementDate(record[75:78])
```

Comment:
Did you want to export this value? Or is your use-case just to read and write the Settlement Date?

Reply:
Good question. I think we would want to get the value at least. Updated the settlementDateField to be public